Crane/app/src/androidTest/java/androidx/compose/samples/crane/calendar/CalendarTest.kt

```diff
@@ -75,48 +74,48 @@ class CalendarTest {
 
     @Test
     fun scrollsToTheBottom() {
-        onNodeWithLabel("January 1").assertExists()
-        onNodeWithLabel("December 31").performScrollTo().performClick()
+        composeTestRule.onNodeWithLabel("January 1", false, false).assertExists()
```

Comment:
Are the `false` params intentional? I found that the `replaceWith` added default values but I think it's clearer without them.

Reply:
Those are actually the default values so not sure why they were there in the first place :) updating! Thanks